### PR TITLE
(changes made) - added python book 

### PR DESCRIPTION
**Pull request by peeplika**
## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?
It is a good book to learn python

### How do we know it's really free?
this book is published under a [Creative Commons Attribution-Share Alike 3.0 license](http://creativecommons.org/licenses/by-sa/3.0/).

### For book lists, is it a book? For course lists, is it a course? etc.
book
## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2008,6 +2008,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Python 2 Official Documentation](https://docs.python.org/2/download.html) (PDF, HTML, TEXT) (2.x)
 * [Python 2.7 quick reference](https://web.archive.org/web/20171013204449/http://infohost.nmt.edu/tcc/help/pubs/python27/python27.pdf) - John W. Shipman (PDF) (2.7) *(:card_file_box: archived)*
 * [Python 3 Official Documentation](https://docs.python.org/3/download.html) (PDF, EPUB, HTML, TEXT) (3.x)
+*  [Python 3 Patterns, Recipes, and Idioms](https://python-3-patterns-idioms-test.readthedocs.io/en/latest/) - Bruce Eckel & Friends
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2011]:
```suggestion
* [Python 3 Patterns, Recipes, and Idioms](https://python-3-patterns-idioms-test.readthedocs.io/en/latest/) - Bruce Eckel & Friends 
```

Comment (eshellman) [reply books/free-programming-books-langs.md:2011]: when you've made changes, please add `(changes made)` to the title of your PR

Comment (peeplika) [reply books/free-programming-books-langs.md]: Thanks for your feedback. I've made the updates mentioned by you

Comment (peeplika) [reply books/free-programming-books-langs.md]: Thanks for your feedback. I've made the updates mentioned by you

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**